### PR TITLE
Update on sensors Feature Policy integration

### PR DESCRIPTION
**Pull request by pozdnyakov**
Update the "Sensors For The Web!" article with the contents about Feature Policy integration in the Generic Sensor API.

### CODE REVIEW

File: src/content/en/updates/2017/09/sensors-for-the-web.md [hunk 1]
```diff
@@ -349,17 +349,24 @@ through HTTPS. During development you can do so via http://localhost but for pro
 you'll need to have HTTPS on your server. See Security with [HTTPS article](/web/fundamentals/security/)
 for best practices and guidelines there.
 
-### Only the main frame
+### Feature Policy integration
 
-To prevent iframes from reading sensor data the Sensor objects can be created
-only within a main frame.
+The [Feature Policy integration](https://w3c.github.io/sensors/#feature-policy-api) in Generic Sensor API is used to control access to sensors data for a frame.
```
Comment (petele) [inline src/content/en/updates/2017/09/sensors-for-the-web.md]: please wrap lines at 80 chars (and throughout)

File: src/content/en/updates/2017/09/sensors-for-the-web.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
-description: Generic Sensor API is available for Origin Trials in Chrome 62.
+description: Generic Sensor API is available for Origin Trials in Chrome 63.
 
 {# wf_updated_on: 2017-09-22 #}
```
Comment (petele) [inline src/content/en/updates/2017/09/sensors-for-the-web.md]: Please update the `{# wf_updated_up #}` tag

Comment (pozdnyakov) [reply src/content/en/updates/2017/09/sensors-for-the-web.md]: done

File: src/content/en/updates/2017/09/sensors-for-the-web.md (hunk 1 above)
Comment (pozdnyakov) [reply src/content/en/updates/2017/09/sensors-for-the-web.md]: done

**Pull request merged**